src/search/operator_counting/operator_counting_heuristic.cc

```diff
@@ -102,6 +103,16 @@ static shared_ptr<Heuristic> _parse(OptionParser &parser) {
     parser.add_list_option<shared_ptr<ConstraintGenerator>>(
         "constraint_generators",
         "methods that generate constraints over operator counting variables");
+
+    parser.add_option<bool>(
+        "use_integer_operator_counts",
+        "restrict operator counting variables to integer values. Computing the "
+        "heuristic with integer variables can produce higher values but "
+        "requires solving an MIP instead of an LP which is generally more "
```

Comment:
-> a MIP

Reply:
That depends on whether you say "mip" or "em-eye-pee". I was going for the second one in analogy to LP ("el-pee"). But I'm fine with changing it as well.

Reply:
That pronounciation didn't occur to me :-) Both options are fine for me.

Reply:
> That pronounciation didn't occur to me :-) Both options are fine for me.

I think "mip" as a word is much more common than em-eye-pee. So I'd go with Jendrik's original suggestion.